### ISSUE TITLE
Is Request-Reply for CONF applications working on 0.3.2?

### ISSUE THREAD

**Issue by gitsaycat** [eclipse/kapua#1940]
I have set up a simple test for Request-Reply CONF application with node-red.
My birth certificate works correctly and I see the 'Device' registered on Kapua devices tab.
As part of the birth certificate I have declared a CONF application and that correctly enables the 'Configuration' tab 
I have also set up a subscription on **

> $EDC/account-name/client-id/CONF-V1/GET/configurations

** 

I was expecting to get REQUEST messages sent to this topic when I click on the 'Configuration' tab. However, no messages are pushed to that topic .
Am I misunderstanding the Request-Reply protocol?
Thanks in advance for information on this.

**Comment by Coduz** [comment]
Hi @gitsaycat ,

just tried on a local Deployment of Kapua 0.3.2 and it works.

The log on the Kura devices is:
```
2018-08-27 06:53:51,765 [MQTT Call: {clientId}] INFO  o.e.k.c.c.CloudServiceImpl - Message arrived on topic: $EDC/kapua-sys/{clientId}/CONF-V1/GET/configurations
2018-08-27 06:53:51,854 [pool-3-thread-2] INFO  o.e.k.c.d.DataServiceImpl - Storing message on topic :$EDC/#account-name/KapuaPool-1535378031262-6844603388585269248/CONF-V1/REPLY/4845442090858401119, priority: 1
2018-08-27 06:53:51,879 [pool-3-thread-2] INFO  o.e.k.c.d.DataServiceImpl - Stored message on topic :$EDC/#account-name/KapuaPool-1535378031262-6844603388585269248/CONF-V1/REPLY/4845442090858401119, priority: 1
2018-08-27 06:53:51,881 [DataServiceImpl:Submit] INFO  o.e.k.c.d.t.m.MqttDataTransport - Publishing message on topic: $EDC/kapua-sys/KapuaPool-1535378031262-6844603388585269248/CONF-V1/REPLY/4845442090858401119 with QoS: 0
```
There are any logs on the broker saying something like:
```
13:57:19.183 [ActiveMQ NIO Worker 1] WARN  o.a.a.t.m.s.AbstractMQTTSubscriptionStrategy - Error subscribing to $EDC/KAPUA-sys/{clientId}/#
java.lang.SecurityException: User 1:kapua-broker ({clientId} - tcp://172.17.0.1:50448 - conn id 2056081308669510746) is not authorized to read from: topic://VirtualTopic.$EDC.KAPUA-sys.{clientId}.>
```

Regards, 

_Alberto_

**Comment by Coduz** [comment]
Hi @gitsaycat ,

I'm closing this due to inactivity. 
If you still need help, please feel free to reopen this issue.

Regards, 

_Alberto_

**Comment by gitsaycat** [comment]
Albert - Hi my apologies for the silence - I have indeed tested and
confirmed that at least part of the functionality works.
My confusion arose from the fact that there are dedicated applications for
some request-response functionality (such as configuration maintenance) and
I was not able to set the names correctly initially.
However, now with the application ids set correctly, Kapua is correctly
enabling certain tabs
Unfortunately, now Kapua does not seem to be talking to those applications
as expected.

I have tested with my own application as well as with Kura clients and I
cannot see configurations being listed in the Device configurations tab
Any help would be appreciated
Thanks!
